### ISSUE TITLE
[Request] Promotion timer

### ISSUE THREAD

**Issue by Biofusionz** [ForgeEssentials/ForgeEssentials#1494]
Something added to permissions so they get promoted to a new group after spending so long in a certain group, eg: BioFusionz is a newbie, been active for a week and get's promoted to a hermit

Or, when a user obtains a certain item. So when a user has steel in their inventory, it promotes them to a new group

**Comment by luacs1998** [comment]
@olee I remember you saying you wanted to bring autopromote back?

**Comment by evernife** [comment]
I will be nice to have "In game Time" not time in real days. (Or maybe, both :D )

But it is a nice ideia.

**Comment by olee** [comment]
I will think about a system to allow this kind of feature.
Basic promotion has been added already but automatic promotion is a bit more complicated.

@Biofusionz is that a urgent matter, oOr can I put this on a bit lower priority?

**Comment by LassyKongo** [comment]
Been looking for this feature, sad to see its not in yet. Tis a very useful thing

**Comment by TimB1087** [comment]
I highly recommend this feature as well. We are trying to implement a greylist, but it's cumbersome since staff need to promote players manually.

**Comment by DPxKnightxZero** [comment]
Just remember each feature added is another chance for bugs.

**Comment by olee** [comment]
Not if I code it :smile: 
I will try to add this in one of the next builds...

**Comment by DPxKnightxZero** [comment]
olee your awesome!

**Comment by luacs1998** [comment]
You could probably do this with cron scripting.

**Comment by olee** [comment]
I still think this is a valid request and should be provided sometime as an actual FE feature.

**Comment by Techjar** [comment]
Can't this be done with the /delayedaction command?

**Comment by luacs1998** [comment]
Nope, not the delayedaction command, but you could check if the conditions are met through cron scripting.

**Comment by olee** [comment]
Yes - with many hacking you can indeed do this with scripting - but that is so complicated that it's not suited for such a feature.
